diff --git a/node_modules/timarney-module3/index.js b/node_modules/timarney-module3/index.js
index 21393ec..529542f 100644
--- a/node_modules/timarney-module3/index.js
+++ b/node_modules/timarney-module3/index.js
@@ -1,7 +1,7 @@
 const flatten = require("lodash.flatten");
 const arr = [1, [1, 2], 3, 4];
 
-const moduleName = "module 3";
+const moduleName = "module 3 - patched";
 
 const print = () => {
   console.log(`== ${moduleName} ==`);
